sharepoint/sharepoint-ps/sharepoint-online/Get-SPOSiteFileVersionExpirationReportJobProgress.md

```diff
@@ -0,0 +1,82 @@
+---
+external help file: sharepointonline.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version: https://learn.microsoft.com/powershell/module/sharepoint-online/get-spositefileversionexpirationreportjobprogress
+applicable: SharePoint Online
+title: Get-SPOSiteFileVersionExpirationReportJobProgress
+schema: 2.0.0
+author: msjennywu
+ms.author: jennywu
+ms.reviewer:
+manager: seanmc
+---
+
+# Get-SPOSiteFileVersionExpirationReportJobProgress
+
+## SYNOPSIS
+
+> [!NOTE]
+> This feature is part of the version history controls preview. If your tenant is not part of the preview or the feature has not rolled out to your tenant, you will get an error when trying to run this cmdlet.
+
+Gets the status for a file version expiration report generation job for a site collection.
+
+## SYNTAX
+
+```powershell
+Get-SPOSiteFileVersionExpirationReportJobProgress [-Identity] <SpoSitePipeBind> [-ReportUrl <String>] [<CommonParameters>]
+```
+
+## DESCRIPTION
+
+Gets the status for a file version expiration report generation job for a site collection.
```

Comment:
Tracks progress of the job to generate version storage report for a site collection.